i18n/fr.yaml

```diff
@@ -5,74 +5,62 @@ twitter_description:
 server_title:
-  other: "Server"
+  other: "Serveur"
 server_content:
-  other: "The server securely collects and stores random IDs uploaded from the mobile app after a confirmed positive test result. It also generates unique temporary codes which grant permission for people with a positive COVID-19 test result to upload their random IDs. The server is flexible enough to be deployed at various levels of government."
+  other: "Le serveur recueille et stocke de façon sécurisée les identifiants aléatoires importés depuis l'application mobile suite à un résultat de test positif confirmé. Il génère également des codes temporaires uniques qui accordent aux individus testés positifs au COVID-19 l'autorisation de mettre en ligne leurs identifiants aléatoires. Le serveur est suffisamment flexible pour un déploiement à différents ordres de gouvernement."
 web_portal_title:
-  other: "Web portal"
+  other: "Portail web"
 web_portal_content_1:
-  other: "The web-based results portal provides health care professionals unique temporary codes which can be shared with users who have tested positive for COVID-19. This code gives a user access to upload their random IDs via the mobile app if they choose. No personal information is collected and there is no association between the codes and specific tests."
+  other: "Le portail web de résultats fournit aux professionnels de santé des codes temporaires uniques qui peuvent être partagés avec les utilisateurs ayant été testés positifs au COVID-19. Ces codes donnent à l'utilisateur un accès lui permettant de mettre en ligne ses identifiants aléatoires via l'application mobile, s'il le souhaite. Aucun renseignement personnel n'est recueilli et il n'existe aucune association entre les codes et des tests particuliers."
 web_portal_content_2:
-  other: "Use of the portal is optional, as the server can also be used to generate temporary COVID Shield codes."
+  other: "L'utilisation du portail est facultative, puisqu'il est aussi possible de recourir au serveur pour générer des codes COVID Shield temporaires."
 who_built_title:
-  other: "Who built COVID Shield?"
+  other: "Qui a conçu COVID Shield ?"
 who_built_content_1:
-  other: "We are a group of Shopify volunteers who want to help to slow the spread of COVID-19 by offering our skills and experience developing scalable, easy to use applications. We are releasing COVID Shield free of charge with a flexible open-source license."
+  other: "Nous sommes un groupe de bénévoles Shopify, et nous souhaitons contribuer au ralentissement de la pandémie de COVID-19 en mettant nos compétences et notre expérience au service du développement d'applications évolutives et conviviales. Nous lançons COVID Shield gratuitement, avec une licence open source flexible."
```

Comment:
**1-** "bénévoles Shopify" is not clear, I would suggest "bénévoles **de chez** Shopify"
**2-** We decided to use "transmission" to translate "spread" instead of "pandemic"
**3-** we say "de la" COVID-19, and not "de" COVID-19
so this paragraph becomes:
`Nous sommes un groupe de bénévoles de chez Shopify, et nous souhaitons contribuer au ralentissement de la transmission de la COVID-19 en mettant nos compétences et notre expérience au service du développement d'applications évolutives et conviviales. Nous lançons COVID Shield gratuitement, avec une licence open source flexible.`

Reply:
Can (should) `open source` be italicised?  🤔

Reply:
I've seen it in the [Larousse](https://www.larousse.fr/dictionnaires/francais/open_source/188163?q=open+source#11058413)? We could translate it by "libre de droit" or "à code ouvert". I guess if the other translator didn't translate it it's because it's intentional to keep an English-tech word?

Reply:
Yes. Hence my suggestion to italicise. Otherwise, we could also put it between french quotes as so: « open source »